internal/cli/iam/organizations/apikeys/accesslists/access_lists.go

```diff
@@ -36,3 +42,68 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+// shouldUseWhitelist returns true when it is service Ops Manager and all versions below 5.0
+// returns false when Atlas, Cloud Manager or Ops Manager 5+.
+func shouldUseWhitelist(s store.ServiceVersionGetter) (bool, error) {
```

Comment:
[nit] can we reverse the name please? 

```suggestion
// shouldUseAccessList returns true when it is service Ops Manager and all versions below 5.0
// returns false when Atlas, Cloud Manager or Ops Manager 5+.
func shouldUseAccessList(s store.ServiceVersionGetter) (bool, error) {
```